### PR TITLE
Add release notes for 0.25.0

### PR DESCRIPTION
**Pull request by SueChaplain**
- Update notable changes table

[ci skip]

Signed-off-by: SueChaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@pshipton - I wasn't sure whether to remove the 2nd limitation about non-compressed builds not being available on AIX or whether this needs to be changed to say that the single build cannot run in non-compressed refs mode. 

Also wondered what to put in the versions column for the z/OS change. In theory it should apply to all versions?

**Comment by pshipton** [comment]
> I wasn't sure whether to remove the 2nd limitation about non-compressed builds not being available on AIX or whether this needs to be changed to say that the single build cannot run in non-compressed refs mode.

The limitation can be removed. The AIX single build supports non-compressed refs just like all other platforms.

> Also wondered what to put in the versions column for the z/OS change. In theory it should apply to all versions?

Yes it does.

File: doc/release-notes/0.25/0.25.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+<td valign="top">z/OS&reg; support for the <strong>%sysname</strong> dump token</td>
+<td valign="top">All versions</td>
+<td valign="top">The <strong>%sysname</strong> dump token is added on z/OS, which equates to the <strong>SYSNAME sysparm</strong>.</td>
+</tr>
+</table>
+
+
+## Known Issues
+
+The v0.24.0 release contains the following known issues and limitations:
```
Comment (PascalSchumacher) [inline doc/release-notes/0.25/0.25.md:84]: I guess this should be `0.25.0`?

Comment (SueChaplain) [reply doc/release-notes/0.25/0.25.md]: Thanks for spotting @PascalSchumacher - I've pushed a fix.

**Review by pshipton — APPROVED** [review]

**Pull request merged**